### ISSUE TITLE
Improve error message(s)

### ISSUE THREAD

**Issue by abremges** [CAMI-challenge/AMBER#30]
I ran AMBER using a corrupt gold standard file, where `_LENGTH` was in the header, but the data rows only contained 2 fields. Example:
```
@Version:0.9.1
@SampleID:gsa

@@SEQUENCEID    BINID    _LENGTH
A    42
B    13
```
I got a plain `IndexError: list index out of range` message and therefore suggest that you check that the data rows indeed contain either 2 or 3 fields and print a sensible error message.

---

For bonus points, I suggest that you check for more and similar edge cases, where better reporting will aid the user in finding the mistake and ultimately a better user experience.

P.S. Eventually, I figured out my mistake (faulty regex removed the `BINID` field) and AMBER worked like a charm! 👍

**Comment by fernandomeyer** [comment]
AMBER now exits with the message below. Fixed in version 1.0.2.

Value in column _LENGTH could not be read. Please provide a value or remove column altogether (and provide a FASTA or FASTQ file instead - see README).
Error. File <filename> is malformed.

**Comment by fernandomeyer** [comment]
Error. File **filename**  is malformed.
